### DISCUSSION TITLE
I love this plugin! May I get to know how to custom the format of the image generated by rectangular selection in my notes?

### DISCUSSION THREAD

**Discussion by IvanRo39** [RyotaUshio/obsidian-pdf-plus#96] [Q&A] [answered]
Well, it's a really amazing plugin, I'm trying to use it to replace the workflow I'm taking on  Marginnote. but I don't know how to custom the format of the image generated by rectangular selection in my notes. Like the screenshot down below, I wonder how to delete the line 103 and the line 105, where both of them contain nothing. Thx :D 
<img width="449" alt="截屏2024-03-09 23 04 48" src="https://github.com/RyotaUshio/obsidian-pdf-plus/assets/162709874/932b1a24-d54a-416d-9ac6-8daf3a471c98">

**Comment by RyotaUshio** [comment, marked as answer]
Thanks! I'm glad PDF++ worked for you.

Rectangular selection is a pretty new feature, so currently you can't customize the format. But I'll enable it in the near future, just like display text formats and link copy formats.

**Reply by IvanRo39** [reply]
Wow, Thx and I'm looking forward to it :D
